src/Loading/babylon.sceneLoader.ts

```diff
@@ -1,16 +1,27 @@
 ﻿module BABYLON {
+    export interface ISceneLoaderPluginExtensions {
+        [extension: string]: {

```

Comment:
That just the definition of a hash table. It means the key is a string and the value is another hash table with a isBinary property.